WordPress/gutenberg
# Cover: Correct resizeable spelling to US resizable

## What?

Updates "resizeable" to the US spelling, "resizable". Updates code comment, file name and related import.

## Why?

Aligns the recently refactored file with the preference for US spelling.

## How?

Fixed spelling. Updated import.

## Testing Instructions
1. Edit a post, add a cover block and resize it via drag handles
2. Ensure no errors
3. Proofread changes.
